src/functions/template-tags/general.php

```diff
@@ -1051,6 +1051,13 @@ function tribe_events_event_schedule_details( $event = null, $before = '', $afte
 			return '';
 		}
 
+		static $cache_details = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: a959b4d632276a7288a7fbdddc39aabe</sub>